src/_components/table_schedule.liquid

```diff
@@ -0,0 +1,37 @@
+<table class="table-schedule">
+  <thead>
+    <tr>
+      <th class="table-schedule__time">Time Slot</th>
+      <th class="table-schedule__event">Talk and Speaker Information</th>
+    </tr>
+  </thead>
+  <tbody>    
+    {% for event in events %}
+      <tr class="table-schedule__slot">
+        <td class="table-schedule__time">
+          <span>{{ event.start_at }}</span>
```

Comment:
maybe bold/enlarge the start time?

Reply:
Maybe 🤔 I am not sure what it would look like, especially on mobile phone screens (that's what most people will use during the event).